documentation/modules/known-issues-2-8.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="known-issues-2-8_{context}"]
+= Known issues
+
+{project-first} 2.8 has the following known issues:
+
+.Target VMware virtual machines lack connectivity as static IP address is not assigned to new network interface
+After migrating Ubuntu 20.04 VMware virtual machines (VM) to {virt} 4.15.6 using {project-short}, the  network interface card (NIC) name of the source VM is not preserved. This results in lack of network connectivity for the VM because the network interface gets a new name post migration. The static IP address that you configured in the network plan before migration is not assigned to the new network interface. link:https://issues.redhat.com/browse/MTV-1607[(MTV-1607)]
```

Comment:
Suggestions: 
Line 10: use "because" instead of "as" in title. 
Line 11: "....machines (VMs)..."  "This results in a lack of..." "gets a new name after the migration." "The static IP address that was configured..." [yes, I am recommending passive voice because the person debugging the problem might not be the same person as the one who created it]